src/openrct2/paint/tile_element/Paint.Surface.cpp

```diff
@@ -657,7 +657,7 @@ static void viewport_surface_draw_tile_side_bottom(
         {
             uint32_t image_id = base_image_id + image_offset;
             PaintAddImageAsParent(
-                session, image_id, offset.x, offset.y, bounds.x, bounds.y, 15, curHeight * COORDS_Z_PER_TINY_Z);
+                session, image_id, { offset.x, offset.y, curHeight * COORDS_Z_PER_TINY_Z }, { bounds.x, bounds.y, 15 });
```

Comment:
```suggestion
                session, image_id, { offset, curHeight * COORDS_Z_PER_TINY_Z }, { bounds, 15 });
```
I think this will work, I'm not sure.

Reply:
I'm against such implicit conversations, this looke like its constructing a 2 component vector that way. I'm not a fan that this is all over the code base honestly. I eventually want to have something like https://github.com/OpenLoco/OpenLoco/pull/809

Reply:
Why do you keep bringing this up? We have discussed this so many times already.